protocols/kad/CHANGELOG.md

```diff
@@ -140,7 +140,7 @@
 
 - Derive `Clone` for `KademliaEvent` (see [PR 2411])
 
-- Derive `Serialize`, `Deserialize` for `store::record::Key` (see [PR 2408])
+- Derive `Serialize`, `Deserialize` for `store::record_priv::Key` (see [PR 2408])
```

Comment:
We shouldn't change old changelogs.

Reply:
@thomaseizinger Yes of course I realized it afterwards, I had done a global replace, then I corrected it immediately, sorry